packages/ui/v2/core/Tips.tsx

```diff
@@ -125,3 +129,82 @@ export default function Tips() {
     </div>
   );
 }
+
+export function TipBanner() {
+  const { t } = useLocale();
+  const [animationRef] = useAutoAnimate<HTMLDivElement>();
+
+  const tips = [
+    {
+      id: 1,
+      img: "/team-banner-background-small.jpg",
+      title: t("calcom_is_better_with_team"),
+      subtitle: t("add_your_team_members"),
+      href: `${WEBAPP_URL}/settings/teams/new`,
+      learnMoreHref: "https://go.cal.com/teams-video",
+    },
+  ];
+
+  // TODO: Refactor to reuse above code and not duplicate function
+  const [list, setList] = useState<typeof tips>([]);
+
+  const handleRemoveItem = (id: number) => {
+    setList((currentItems) => {
+      const items = localStorage.getItem("removedBannerIds") || "";
+      const itemToRemoveIndex = currentItems.findIndex((item) => item.id === id);
+
+      localStorage.setItem(
+        "removedBannerIds",
+        `${currentItems[itemToRemoveIndex].id.toString()}${items.length > 0 ? `,${items.split(",")}` : ""}`
+      );
+      currentItems.splice(itemToRemoveIndex, 1);
+      return [...currentItems];
+    });
+  };
+
+  useEffect(() => {
+    const reversedTips = tips.slice(0).reverse();
+    const removedTipsString = localStorage.getItem("removedBannerIds") || "";
+    const removedTipsIds = removedTipsString.split(",").map((id) => parseInt(id, 10));
+    const filteredTips = reversedTips.filter((tip) => removedTipsIds.indexOf(tip.id) === -1);
+    setList(() => [...filteredTips]);
+  }, []);
```

Comment:
can someone help me refactor this? this should be merged with the above identical function.